frontend/src/features/projects/common/components/FilterBar.tsx

```diff
@@ -44,26 +46,22 @@ export interface IFilterBarState {
 }
 
 type FilterBarProps = {
-  agencyLookupCodes: ILookupCode[];
-  propertyClassifications: ILookupCode[];
   onChange: (value: IFilterBarState) => void;
 };
 
 /**
  * Filter bar for the Property List view
  */
-const FilterBar: React.FC<FilterBarProps> = ({
-  agencyLookupCodes,
-  propertyClassifications,
-  onChange,
-}) => {
-  const mapLookupCode = (code: ILookupCode): SelectOption => ({
-    label: code.name,
-    value: code.id.toString(),
-  });
+const FilterBar: React.FC<FilterBarProps> = ({ onChange }) => {
+  const lookupCode = useCodeLookups();
   //restrict available agencies to user agencies.
-  const agencies = (agencyLookupCodes ?? []).map(c => mapLookupCode(c));
-  const classifications = (propertyClassifications ?? []).map(c => mapLookupCode(c));
+  const agencies = lookupCode.getOptionsByType(API.AGENCY_CODE_SET_NAME);
+  const classifications = lookupCode.getPropertyClassificationOptions(
```

Comment:
this does make the assumption that this filter bar should never use these values regardless of where it is used, I'm fine with this because of where the project is at.

Reply:
Also doesn't getPropertyClassificationOptions already do this?

Reply:
`getPropertyClassificationOptions(...)` will by default only return options that your claims allow.  However I've extended it to allow for a custom filter when required.  There are a few places where it is easier to implement if we have full control of the results.